jetty-server/src/main/java/org/eclipse/jetty/server/SecureRequestCustomizer.java

```diff
@@ -377,4 +400,40 @@ public Object getAttribute(String name)
             return names;
         }
     }
+
+    /**
+     * Simple bundle of information that is cached in the SSLSession.
+     */
+    private static class SslSessionData
+    {
+        /**
+         * The name of the SSLSession attribute that will contain this information.
+         */
+        private static final String ATTR = SslSessionData.class.getName();
+        private final Integer _keySize;
+        private final X509Certificate[] _certs;
+        private final String _idStr;
+
+        private SslSessionData(int keySize, X509Certificate[] certs, String idStr)
+        {
+            this._keySize = keySize;
+            this._certs = certs;
+            this._idStr = idStr;
+        }
+
+        private X509Certificate[] getCerts()
```

Comment:
Put this getter after `getKeySize()` to be consistent with ordering.

Reply:
Moved.